src/components/FormControl/FormControl.test.tsx

```diff
@@ -0,0 +1,38 @@
+import { render, screen } from '@testing-library/react';
+import React from 'react';
+
+import FormControl from './FormControl';
+
+describe('<FormControl />', () => {
+	it('Should be able to render children', () => {
+		const child = '123';
+		render(<FormControl>{child}</FormControl>);
+
+		expect(screen.getByText(child)).toBeDefined();
```

Comment:
Ik denk dat `toBeInTheDocument()` hier relevanter is. Ook voor de onderstaande testen